Core/src/org/sleuthkit/autopsy/commonfilesearch/Bundle.properties

```diff
@@ -0,0 +1,7 @@
+CommonFilesNode.getName.text=Common Files
+CommonFilesDialog.frame.title=Find Common Files
+CommonFilesDialog.frame.msg=Find Common Files
+CommonFilesPanel.searchButton.text=Search
+CommonFilesPanel.search.results.title=Common Files
+CommonFilesPanel.search.results.pathText=Common Files Search Results\:
+CommonFilesAction.getName.text=Common Files Search
```

Comment:
Please use @Message annotations for all messages not generated by the IDE UI design tool (Matisse).

Reply:
Could you provide an example, I don't follow?

Reply:
@briangsweeney, is the the SwingWorker code sample I emailed out as SwingWorker_Snippet_2.text sufficient as an example?

Reply:
got it now. thx!